docs/application/native/guides/text-input/autofill.md

```diff
@@ -104,7 +88,7 @@ To get the presence of Autofill data and Autofill authentication information:
 
 1. Create a callback function using the `auth_info_received_cb()` function:
 
-   You need to implement the UI elements to display the name, service message, and logo, image of autofill service, incase the authentication is required. In addition, `autofill_auth_info_request()` to retrieve the data previously entered by the user and stored in an autofill service must be called when pressing UI to show Autofill service information:
+   You need to implement the UI elements to display the name, service message, logo, and image of Autofill service, in case the authentication is required. In addition, the function `autofill_auth_info_request()` must be called when pressing UI to show the Autofill service information, it retrieves the data previously entered by the user that is stored in an Autofill service related storage:
```

Comment:
Suggested change:
It retrieves the data previously entered by the user that is stored in an Autofill service related storage:

Reply:
Fixed.